torchani/data/__init__.py

```diff
@@ -364,6 +365,10 @@ def create_aev_cache(dataset, aev_computer, output, enable_tqdm=True, encoder=la
 
 def _cache_aev(output, dataset_path, batchsize, device, constfile,
                subtract_sae, sae_file, enable_tqdm, encoder, **kwargs):
+
+    warnings.warn("cache_aev() is deprecated; use cache_sparse_aev()",
```

Comment:
Not here, this is called in both `cache_aev` and `cache_sparse_aev`. Move this to the `cache_aev` below.

Reply:
oops! sorry!